base/deprecated.jl

```diff
@@ -788,6 +788,19 @@ function symperm{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}, pinv::Vector{Ti})
         "Pkg.add(\"SuiteSparse\") to install SuiteSparse on Julia v0.5."))
 end
 
+# needs to be a macro so that we can use ::@mime(s) in type declarations
+eval(Multimedia, quote
+export @MIME
+macro MIME(s)
+    Base.warn_once("@MIME(\"\") is deprecated, use MIME\"\" instead.")
```

Comment:
apparently this has been deprecated for almost 3 years, just was never in this file - I think it would be safe to just delete it? https://github.com/JuliaLang/julia/commit/af8c18c1f088e59b3bc0249fd4e4bfd27851606d